packages/wonder-blocks-core/src/__docs__/exports.use-on-mount-effect.stories.mdx

```diff
@@ -0,0 +1,23 @@
+import {Meta} from "@storybook/addon-docs";
+
+<Meta
+    title="Core / Exports / useOnMountEffect()"
+    parameters={{
+        chromatic: {
+            disableSnapshot: true,
+        },
+    }}
+/>
+
+# useOnMountEffect()
+
+```ts
+function useOnMountEffect(callback: (void | () => void)) void;
+```
+
+The `useOnMountEffect` can be used to run an effect once on mount. This avoids
+having to pass `useEffect` an empty deps array and disable the
+`react-hooks/exhaustive-deps` lint.
+
+If `callback` returns a cleanup function, it will be called when the component
+is unmounted.
```

Comment:
**thought:** Based on the example you shared with me. What do you think about expanding the docs here to include an example?

I usually use the `Usage` section for other WB components, so maybe that could work here to explain that. wdyt?

https://github.com/Khan/wonder-blocks/blob/a8deae7200aefaa8c5f69b3ef3db3cae9be2b4fd/packages/wonder-blocks-spacing/src/__docs__/spacing.stories.mdx?plain=1#L33-L43

Reply:
I think it may be useful to also include in the docs what the equivalent `useEffect` code looks like. This would not only help explain to folks what this is doing for them, but it would also help them identify places that could be tidied up by using this hook.